### ISSUE TITLE
cannot edit annotations after reload from database. dwv-jqmobile

### ISSUE THREAD

**Issue by shameerasl** [ivmartel/dwv#1207]
hi.. I am using dwv-jqmobile as my dicom viewer in my project.
I have saved images in database with rectangle draws(annotations). and also read the images and load again in my viewer with the annotations. my code is
```js
myapp.loadURLs([selected_url],[{name:"contentType", value:"image/jpeg"}]);
myapp.setDrawings(drawings,drawingsDetails); 
```
The issue I am facing is I cannot edit the draws.when I click the retreived annotations it is not select. how to resolve this. hope you reply me with references and solutions. thanks in advance.

**Comment by ivmartel** [comment]
It is true the loaded drawing is not selected. In my case, I have to draw another to then be able to select the first one. 
See also the pacs test viewer with the 'Baby MRI (with state)' example for some demo.

**Comment by ivmartel** [comment]
Relates to #1260.
Fixed in 32dba82.
